client/src/components/users/Users.jsx

```diff
@@ -83,39 +86,46 @@ const response = await addUserApi(newUser)
         <thead>
           <tr>{renderHeader()}</tr>
         </thead>
-        <tbody>{renderBody()}</tbody>
+        {renderBody()}
       </table>
 
       <div>
         <h3>Add User</h3>
-        <form id="add-user" data-testid = "user-form" action="#" onSubmit={handleAddOnSubmit}>
+        <form
+          id="add-user"
+          data-testid="user-form"
+          action="#"
+          onSubmit={handleAddOnSubmit}
+        >
           <fieldset>
-            <label>Name</label>
+            <label htmlFor="name">Name</label>
```

Comment:
The `htmlFor` value should match the `id` of the input that you want to associate it with. See the docs for more info: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/for

```suggestion
            <label htmlFor="add-user-name">Name</label>
```